src/test/java/pl/stormit/eduquiz/quizcreator/controllers/QuizApiControllerTest.java

```diff
@@ -56,13 +59,10 @@ void shouldReturn200WhenFoundAllQuizzes() throws Exception {
         given(quizService.getQuizzes()).willReturn(quizzesDtos);
 
         //when
-        ResultActions result = mockMvc.perform(get("/api/v1/quizzes")
-                .contentType(MediaType.APPLICATION_JSON));
+        ResultActions result = mockMvc.perform(get("/api/v1/quizzes").contentType(MediaType.APPLICATION_JSON));
 
         //then
-        result.andExpect(status().isOk())
-                .andExpect(content().string(containsString("PL")))
-                .andExpect(content().string(containsString("UK")));
+        result.andExpect(status().isOk()).andExpect(content().string(containsString("PL"))).andExpect(content().string(containsString("UK")));
```

Comment:
I think the previous assert version is more readable:
result.andExpect(status().isOk())
                .andExpect(content().string(containsString("PL")))
                .andExpect(content().string(containsString("UK")));
               
You can see immediately what we are checking